packages/synapse-react-client/src/components/Plot/DotPlot.tsx

```diff
@@ -38,8 +39,19 @@ function getLayout(
   if (layoutOptions.backgroundColor) {
     result.plot_bgcolor = layoutOptions.backgroundColor
   }
+  result.yaxis!.showgrid = false
+  result.yaxis!.zeroline = true
+  result.yaxis!.zerolinecolor = '#aeb5bc'
   result.yaxis!.showticklabels = false
-  result.xaxis!.range = [-5, layoutOptions.maxValue! + 30]
+  if (result.xaxis!.type === 'log') {
+    result.xaxis!.tickformat = 'f'
+    result.xaxis!.range = [-0.1, Math.log10(layoutOptions.maxValue!) + 0.1]
+  } else {
+    result.xaxis!.range = [
+      (layoutOptions.maxValue! / 50) * -1,
```

Comment:
In non-log mode (which will not be actively used after this change), make the lower bound dynamic based on the max value to prevent dots being cut off.